R/tests.R

```diff
@@ -315,6 +323,11 @@ testPlots <- function() {
         displayName = "scatterplot_multifeat",
         plotType = c("multiFeature", "multiTest"),
         packages = "data.table"
+      ),
+      multiModel_scatterplot = list(
+        displayName = "mmplot",
+        packages = "stats",
+        plotType = "multiModel"
```

Comment:
Shouldn't the plotType also include "multiFeature"? Otherwise it will only contain a single point.

Reply:
As it currently stands, multiModel will return as many features as provided by featureID, irrespective if one of several. I may be missing smth here, but should we make discrete distinction between single- vs. multi-feature? Of course, if some computation (say, PCA) is performed in a set of features, this cannot be done when selecting a single feature, but I presume we can simply control it by plotting in the FE what does not return an error. Or?

Reply:
You're correct that the BE R code will work with any length featureID. However, the FE only calls plots depending on their `plotType`. Since this defaults to `singleFeature`, this plot will never be called by the app with more than one featureID. I think it should be `plotType = c("multiFeature", "multiModel")`. A scatter plot with a single featureID is unlikely to be very informative anyways.